### PR TITLE
chore: updated quill buttons and the package itself

### PR DESCRIPTION
**Pull request by mohsen-deriv**

### CODE REVIEW

File: libs/blocks/src/lib/search/index.tsx [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
+        onChange={(e) => {
+          showSearchResults(e.target.value);
 
           if (onChange) {
-            onChange(value);
+            onChange(e.target.value);
           }
         }}
-        className="w-full "
-        inputClassName="!bg-gradient-slate-110"
```
Comment (prince-deriv) [inline libs/blocks/src/lib/search/index.tsx:29]: Are we handling the control for devices that are using dark theme by default? Is it handled somewhere?

File: libs/templates/src/lib/home/sections/account-section/data.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ export const TableData: TableContent = {
   description:
     'Direct market access with fast order execution and no dealing desk intervention.',
   cta: (
-    <Button variant="secondary" className="w-1/2">
+    <Button variant="secondary" colorStyle="black" size="lg" className="w-1/2">
```
Comment (prince-deriv) [inline libs/templates/src/lib/home/sections/account-section/data.tsx:11]: I think the we need to adjust the props in the quill-design. 

```suggestion
    <Button variant="secondary" color="black" size="lg" className="w-1/2">
```

File: libs/templates/src/lib/home/sections/hero/data.tsx [hunk 3]
```diff
@@ -69,46 +71,29 @@ export const renderHomeHeroSliderContent = () => (
 
 export const HomeHeroForm = () => {
   return (
-    <FluidContainer className={`flex flex-col gap-general-lg pr-general-3xl`}>
-      <div className="flex flex-col justify-between gap-gap-lg md:flex-row">
-        <input
-          placeholder="This component is not styled yet"
-          className="rounded-2xl border-300 border-solid-red-600"
-        />
+    <div className={`flex flex-col gap-general-lg`}>
```
Comment (prince-deriv) [inline libs/templates/src/lib/home/sections/hero/data.tsx:74]:
```suggestion
    <div className="flex flex-col gap-general-lg">
```

**Review by prince-deriv — CHANGES_REQUESTED** [review]

File: libs/blocks/src/lib/search/index.tsx (hunk 1 above)
Comment (mohsen-deriv) [reply libs/blocks/src/lib/search/index.tsx:29]: it should managed by quill-design, I'm gonna have a chat with core design team about this. for now we have to skip it.

File: libs/templates/src/lib/home/sections/account-section/data.tsx (hunk 2 above)
Comment (mohsen-deriv) [reply libs/templates/src/lib/home/sections/account-section/data.tsx:11]: we can't do it, color is a html attribute man. so we can't use it.

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**